ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -771,3 +771,33 @@ CH_SS_CHWSWISOVM_CWSISOVM_CWRISOVM_CHWRWISOVM_CHWDT:
   opt_uses:
   - lost_power_alarm
   - run_mode
+
+CH_SS_SCHWTC_CWDT_REFSM_CHWFRSM_CDWFRSM_CHDXSC_IGM:
+  guid: "38027ed7-d703-4971-a958-5c125fd41fce"
+  description: "Simple water-cooled chiller with supply temperature control with refrigerant monitoring, chilled and condenser water flowrate status monitoring."
+  is_canonical: true
+  implements:
+  - CH
+  - SS
+  - SCHWTC
+  - CWDT
+  - REFSM
+  - CHWFRSM
+  - CDWFRSM
+  - CHDXSC
+  - IGM
+  opt_uses:
+  - compressor_run_time_accumulator
+
+CH_SS_CC2XM_REFSM2X_REFPM2X:
```

Comment:
are there any temperature control points?

Reply:
there is no temperature control points for this

Reply:
It there a setpoint in the bms that is not bacnet visible? I do not understand how the chiller can be not controling to anything.